VideoLocker/src/main/java/org/edx/mobile/util/ResourceUtil.java

```diff
@@ -27,4 +30,22 @@ public static String getResourceString(String name, Context context) {
             return context.getString(nameResourceID);
         }
     }
+
+    public static String getResourceString(int resourceId){
+        Context context = MainApplication.instance().getApplicationContext();
+        return context.getString(resourceId);
+    }
+
+    public static CharSequence getFormattedString(int resourceId, String key, String value){
+        if ( value == null )
+            value = "";
+        return Phrase.from(ResourceUtil.getResourceString(resourceId))
+                .put(key, value) .format();
+    }
+
+    public static CharSequence getFormattedStringForQuantity(int resourceIdForSingle, int resourceIdForPlural,
```

Comment:
Unfortunately this won't work. Different languages may have very different pluralization rules that involve more than two alternatives. See this insane table http://unicode.org/repos/cldr-tmp/trunk/diff/supplemental/language_plural_rules.html